sormas-app/app/src/main/java/de/symeda/sormas/app/util/LocationService.java

```diff
@@ -157,7 +158,7 @@ public boolean isRequestingActiveLocationUpdates() {
 
 	public void requestFreshLocation(Activity callingActivity) {
 
-		if (!validateGpsAccessAndEnabled(callingActivity)) {
+		if (ConfigProvider.getUser() == null || !validateGpsAccessAndEnabled(callingActivity)) {
```

Comment:
I would include the test for user inside the `validateGpsAccessAndEnabled` method snce they are always used together.